abci/ve/utils.go

```diff
@@ -182,10 +182,15 @@ func ValidateVoteExtensions(
 			continue
 		}
 
+		// If the validator does not have a valid public key, we skip the signature verification logic but still include
+		// the validator's voting power in the total voting power. The app may have pruned the validator's public key
+		// from the store, but comet considered the validator as active and included them in the commit since there
+		// is a 1 block delay between the validator set update on the app and comet.
+		sumVP += vote.Validator.Power
 		valConsAddr := sdk.ConsAddress(vote.Validator.Address)
 		pubKeyProto, err := valStore.GetPubKeyByConsAddr(ctx, valConsAddr)
 		if err != nil {
-			return fmt.Errorf("failed to get validator %X public key: %w", valConsAddr, err)
+			continue
```

Comment:
should we have a comment on why we made this change?

Reply:
I think the comment above should cover it. do u think there is anything else i should add to the comment?

Reply:
oh whoops, i missed that lol 😪😪 😪😪